### ISSUE TITLE
Wrong label "Ignore Tab Unload"

### ISSUE THREAD

**Issue by Fxxxlei** [zen-browser/desktop#1990]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

In the tab's context menu under "Tab Unloader Actions" there is an item called "Ignore Tab Unload".
I'm not a native but very fluent English speaker and I had no idea what this menu item does and would even assume it is synonymous to "Prevent Tab Unload". Looking at the code, I am pretty sure this should actually be called "Allow Tab Unload", but correct me if I am wrong.

Alternatively, both items could be summed up as a single checkbox item, though this would be able to have an indeterminate state when multiple tabs are selected.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.7 and twilight

### What platform are you seeing the problem on?

Linux

**Comment by mauro-balades** [comment]
Okay! Fixed in the next update, thanks!

I've changed it to `Prevent Tab Unload`

**Comment by Fxxxlei** [comment]
That was fast, but I meant it should be the opposite:
`Allow Tab Unload`

since the code looks like it's undoing what the already existing `Prevent Tab Unload` does.

**Comment by Fxxxlei** [comment]
@mauro-balades are you sure it should be `Prevent Tab Unload`? we already had that item.

**Comment by mauro-balades** [comment]
Ohh true, I'll change now

**Comment by harrrrshit** [comment]
Hi, please help me out here. 
I want to know the difference between the two options there is in Tabs Unloader Actions -: Prevent Tab Unload & Ignore Tab Unload. 
## I just lost my data on a tab, since the tab unloaded and everything got erased with it.

**Comment by mauro-balades** [comment]
> Prevent Tab Unload

This make it so the tab does NOT get unloaded automatically

> Ignore Tab Unload

Does set it so it unloads automatically. Really sorry for the confusion, already fixed the bug in the next release

**Comment by harrrrshit** [comment]
### Hi, Thanks for the quick explanation, and I am really sorry to disturb you again.

But I again got a problem related to this.

Before the latest update to 1.0.1-a.8, from 1.0.1-a.7.
As you explained me, I was doing `Prevent Tab Unload` to make them not automatically unload.

> > Prevent Tab Unload
> 
> This make it so the tab does NOT get unloaded automatically


But what happened now, after I updated to 1.0.1-a.8, is that both of them now have same name `Prevent Tab Unload & Prevent Tab Unload (I)`.
### Now I don't even know which one is OG `Prevent Tab Unload` which you told me to use so that I can set the tab not to unload automatically.

**Comment by Fxxxlei** [comment]
Yes, there was a small mistake that was fixed within a few hours. It might have happened during release.
All but the English (US) translation have been fixed in the repo, though I think the US version might have been missed @mauro-balades.

Until the next release, the `(I)` is a remainder of the former `Ignore` label, so the item with `(I)` allows unloading, the other one prevents. On that note, we might want to change the accelerator key to `A` to match the new label (just food for thought).

I've gotten more confident in understanding the repos now, so I'll try making pull requests next time, so mauro can concentrate on other things.

**Comment by harrrrshit** [comment]
### Oh Got it, Thanks.

So like which is the native default language @mauro-balades uses??? I switched to English(GB) and this issue is not in there. I might just use English(UK) from now on, since using English(UK) or English(US) doesn't make much of a difference for me.

And BTW, What do you mean by 
> On that note, we might want to change the accelerator key to A to match the new label (just food for thought).

Also, if you don't mind and know where the code is of this thing in the repo, can you please share, I will change and submit a pull request to speed up the process for anyone using English(US).

Edit: I found the Language Packs repo, fixed and submitted a [Pull Request](https://github.com/zen-browser/l10n-packs/pull/72).

**Comment by mauro-balades** [comment]
> I've gotten more confident in understanding the repos now, so I'll try making pull requests next time, so mauro can concentrate on other things.

Id love that!

**Comment by harrrrshit** [comment]
Hi @mauro-balades I have submitted a [pull request](https://github.com/zen-browser/l10n-packs/pull/71) for the same.
